explorer-api-common/src/main/java/org/zowe/api/common/connectors/zosmf/ZosmfConnector.java

```diff
@@ -95,7 +66,75 @@ public HttpResponse request(RequestBuilder requestBuilder) throws IOException {
             throw new ZosmfConnectionException(e);
         }
         return client.execute(requestBuilder.build());
+    }
+
+    /**
+     * Make a Preemptive Basic Authentication HttpClient
+     * @param credentialsProvider is the credential Provider
+     * @return return the httpclient
+     * @throws NoSuchAlgorithmException the exception
+     * @throws KeyManagementException the exception too
+     */
+    public static HttpClient createPreemptiveHttpClientIgnoreSSL(CredentialsProvider credentialsProvider)
+            throws NoSuchAlgorithmException, KeyManagementException {
+        SSLContext sslcontext = SSLContext.getInstance("TLS");
+        sslcontext.init(null, new TrustManager[]{new X509TrustManager() {
+            @Override
+            public void checkClientTrusted(X509Certificate[] arg0, String arg1) {
+            }
+
+            @Override
+            public void checkServerTrusted(X509Certificate[] arg0, String arg1) {
+            }
+
+            @Override
+            public X509Certificate[] getAcceptedIssuers() {
+                return new X509Certificate[0];
+            }
+
+        } } , new java.security.SecureRandom());
+        return HttpClientBuilder.create().setSSLContext(sslcontext).setDefaultCredentialsProvider(credentialsProvider)
+                .setSSLHostnameVerifier(new HostnameVerifier() {
+
+                    @Override
+                    public boolean verify(String s1, SSLSession s2) {
+                        return true;
+                    }
+
+                }).build();
+    }
+
+
+    public static HttpClient createIgnoreSSLClientWithPassword(String userName, String password)
+            throws NoSuchAlgorithmException, KeyManagementException {
+        CredentialsProvider credentialsProvider = new BasicCredentialsProvider();
+        credentialsProvider.setCredentials(AuthScope.ANY, new UsernamePasswordCredentials(userName, password));
+
+        SSLContext sslcontext = SSLContext.getInstance("TLS");
+        sslcontext.init(null, new TrustManager[]{new X509TrustManager() {
+            @Override
+            public void checkClientTrusted(X509Certificate[] arg0, String arg1) {
+            }
+
+            @Override
+            public void checkServerTrusted(X509Certificate[] arg0, String arg1) {
+            }
+
+            @Override
+            public X509Certificate[] getAcceptedIssuers() {
+                return new X509Certificate[0];
+            }
+
+        } } , new java.security.SecureRandom());
+        return HttpClientBuilder.create().setSSLContext(sslcontext).setDefaultCredentialsProvider(credentialsProvider)
+                .setSSLHostnameVerifier(new HostnameVerifier() {
+
+                    @Override
+                    public boolean verify(String s1, SSLSession s2) {
+                        return true;
+                    }
 
+                }).build();
```

Comment:
I think these methods are only used in the LTPA version, so can be moved to the subclass